packages/moonstone/Dropdown/Dropdown.js

```diff
@@ -80,10 +80,15 @@ const DropdownList = Skinnable(
 			/**
 			 * The selections for Dropdown
 			 *
-			 * @type {String[]}
+			 * @type {String[]|Object[]}
```

Comment:
@ryanjduffy What should we use to indicate the object must have a `key` prop?   `Array.<{key}>`?  `Array.<{key: Number|String}>`?

Reply:
the first looks okay to me if it renders int the docs well. I don't think we need to maintain the specific type of key in our docs unless necessary

Reply:
I was thinking for TypeScript generation.

Reply:
Ah. The latter would be more correct in that regard. Not sure it's critical but maybe not too painful since we only reference it a couple spots.

Reply:
I'll test it to see what drops out.